library/kotlin/src/io/envoyproxy/envoymobile/stats/Element.kt

```diff
@@ -0,0 +1,18 @@
+package io.envoyproxy.envoymobile
+
+import java.util.regex.Pattern
+
+/**
+ * Element represents one dot-delimited component of a time series name.
+ *
+ * Element values must conform to the regex /^[A-Za-z_]+$/.
+ */
+class Element(val element: String) {
+  init {
+    if (!Pattern.compile("^[A-Za-z_]+\$").matcher(element).matches()) {
+      throw IllegalArgumentException(
+        "Element values must conform to the regex /^[A-Za-z_]+$/"
```

Comment:
Regex is hardcoded 3 places here. Can we use a constant instead so we don't accidentally fail to change one of the usages in the future?

Reply:
fair